lib/qless/job.rb

```diff
@@ -63,6 +63,9 @@ def perform
         return fail("#{queue_name}-NameError", "Cannot find #{klass_name}")
       end
 
+      # log a real process executing job -- before we start processing
+      log("started by pid:#{Process.pid}")
```

Comment:
Rather, this seems like it would be widely-useful, but should probably be opt-in.